cli/azd/pkg/project/service_target_aks.go

```diff
@@ -297,7 +297,7 @@ func (t *aksTarget) deployManifests(
 	ctx context.Context,
 	serviceConfig *ServiceConfig,
 	task *async.TaskContextWithProgress[*ServiceDeployResult, ServiceProgress],
-) (*kubectl.Deployment, error) {
+) (bool, *kubectl.Deployment, error) {
```

Comment:
Consider adding a note to the doc comment that describes what this bool means (which I think it means "we submitted the manifest to the k8s control plane and it was accepted, even if we fail later while trying to wait for the deployment to complete).